packages/thirdweb/src/extensions/erc1155/token1155.test.ts

```diff
@@ -37,12 +38,21 @@ describe.runIf(process.env.TW_SECRET_KEY)("TokenERC1155", () => {
       chain: ANVIL_CHAIN,
       client: TEST_CLIENT,
       params: {
-        name: "Test TokenERC1155",
-        contractURI: TEST_CONTRACT_URI,
+        name: "Edition",
```

Comment:
Same here. Also this tests the exact same thing no? Doesn't actually help coverage

Reply:
added back